src/components/Layout/ClaimRewardButton/index.tsx

```diff
@@ -111,6 +111,9 @@ export const ClaimRewardButton: React.FC<ClaimRewardButtonProps> = props => {
     uncheckedGroupNames,
   });
 
+  // TODO: wire up (VEN-932)
+  const handleClaimReward = async () => fakeContractReceipt;
```

Comment:
@therealemjy should we update this now?

Reply:
I'll do this in a separate PR which will use this work, otherwise this one will get too big